iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -180,24 +180,21 @@ class ConversationListViewController: ViewController,
         }
     }
 
-    func updateCenterMostCell() {
-        guard let ip = self.collectionView.centerIndexPath(),
-              let conversation = self.conversationListController.conversations[safe: ip.item] else { return }
-
-        /// Sets the active conversation
-        ConversationsManager.shared.activeConversation = conversation
-
-        let members = conversation.lastActiveMembers.filter { member in
-            return member.id != ChatClient.shared.currentUserId
-        }
-        self.messageInputAccessoryView.textView.setPlaceholder(for: members, isReply: false)
+    func update(withCenteredConversation cid: ConversationId?) {
```

Comment:
Changing this function to take in the conversation id of the centered conversation.